flow/connectors/postgres/postgres.go

```diff
@@ -600,27 +600,19 @@ func (c *PostgresConnector) GetTableSchema(
 
 func (c *PostgresConnector) getTableSchemaForTable(
 	tableName string,
-	skipPkeyAndReplicaCheck bool,
 ) (*protos.TableSchema, error) {
 	schemaTable, err := utils.ParseSchemaTable(tableName)
 	if err != nil {
 		return nil, err
 	}
 
-	var pKeyCols []string
-	var replicaIdentityType ReplicaIdentityType
-	if !skipPkeyAndReplicaCheck {
-		var replErr error
-		replicaIdentityType, replErr = c.getReplicaIdentityType(schemaTable)
-		if replErr != nil {
-			return nil, fmt.Errorf("[getTableSchema]:error getting replica identity for table %s: %w", schemaTable, replErr)
-		}
-
-		var err error
-		pKeyCols, err = c.getPrimaryKeyColumns(replicaIdentityType, schemaTable)
-		if err != nil {
-			return nil, fmt.Errorf("[getTableSchema]:error getting primary key column for table %s: %w", schemaTable, err)
-		}
+	replicaIdentityType, err := c.getReplicaIdentityType(schemaTable)
+	if err != nil {
+		return nil, fmt.Errorf("[getTableSchema] error getting replica identity for table %s: %w", schemaTable, err)
+	}
+	pKeyCols, err := c.getPrimaryKeyColumns(replicaIdentityType, schemaTable)
+	if err != nil {
+		return nil, fmt.Errorf("[getTableSchema] error getting primary key column for table %s: %w", schemaTable, err)
```

Comment:
When a user kicks off an initial_copy_only mirror via query layer, the mirror will fail here for tables with no pkey/replica identity

Reply:
changed getPkeyColumns to not error out